pkg/operator/controller/status/controller.go

```diff
@@ -140,7 +140,14 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 
 	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorAvailableCondition(allIngressesAvailable))
 	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorProgressingCondition(allIngressesAvailable, oldStatus.Versions, co.Status.Versions, r.OperatorReleaseVersion, r.IngressControllerImage))
-	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorDegradedCondition(state.IngressControllers))
+	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorIngressDegradedCondition(state.IngressControllers))
+
+	authStateObj := &operatorv1.Authentication{ObjectMeta: metav1.ObjectMeta{Name: "cluster"}}
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: "authentication"}, authStateObj); err != nil {
+		log.Error(err, "unable to get authentication cluster operator")
```

Comment:
Fail fast/return early on failure

Reply:
my intention was not to affect ingress operator when auth operator object does not exists... iow. do not create unnecessary dependency

Reply:
If it's worth logging an error for, should we publish a status that indicates the auth operator is broken in this case? 😁

If it's fine to ignore, why log anything? Maybe just publish a warning event to reduce log spam?

Reply:
@ironcladlou i think this is just log error worthy, we don't need event spam when this happen as this is pretty rare, but it is good to have some trace in log if things get weird or this stop working (for debugging)... I can use higher verbosity if you like?